### PR TITLE
[prod-stable] update chrome

### PR DESCRIPTION
**Pull request by Hyperkid123**

### CODE REVIEW

**Review by fhlavac — APPROVED** [review]

**Review by karelhala — APPROVED** [review]

**Comment by Hyperkid123** [comment]
I would like to wait for fo @john-dupuy approval on the prod beta version.

**Review by john-dupuy — APPROVED** [review]
Based on results of our `prod-test-suite` (which runs against prod-beta), this LGTM. 

FYI @rsnyman @jjaquish

**Pull request merged**
